platform/linux-generic/odp_packet.c

```diff
@@ -2607,6 +2635,8 @@ int odp_packet_parse(odp_packet_t pkt, uint32_t offset,
 	int ret;
 	uint16_t ethtype;
 	uint64_t l4_part_sum = 0;
+	const uint32_t min_seglen = PARSE_ETH_BYTES + PARSE_L3_L4_BYTES;
```

Comment:
Quite a lot of macroing to come up with one constant value. The alternative would be to just put the value here and comment where it comes from, but I'm sure you already considered that. Anyway, this is fine.

Reply:
Yes, it is not very nice, but I did not see a better way. I think a single constant would be easily forgotten when things change. With the macros defined near the relevant functions I hope that the constraints stay up to date better.